### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by AC159**
# General info

**Issue number**: #13 #18 

**Task description**: 

 - Implemented routes to raise flag on patients and get patient covid info by healt and immigration officials
 - Change 2

# Testing

**Test coverage**:
Tests are covering the newly implemented routes

# Additional notes

**Note to reviewers**:

**To do before merging**:
- [ ] Edit Changelog(if necessary)

### CODE REVIEW

File: repository/PatientRepository.js [hunk 1]
```diff
@@ -22,6 +33,15 @@ class PatientRepository {
   fetchPatientStatusForms(userId) {
     return this.mongo.db('test').collection('patientForms').find({patientUid: userId}).toArray();
   }
+
+  raiseFlag(userId, flagType, flagValue) {
+    return this.mongo.db('test').collection('user').updateOne({uid: userId}, {$set: {flagType: flagValue}});
+  }
+
+  fetchPatientsCovidInfo() {
+    return this.mongo.db('test').collection('user').find({userType: 'patient', userStatus: 'APPROVED'}, {_id: 0, uid: 1, covidStatus: 1, name: 1}).toArray();
```
Comment (mth-1012) [inline repository/PatientRepository.js:42]: Add covidStatus to the **_data_** inside route /addnewUser and I think we're good

**Review by mth-1012 — APPROVED** [review]
Good work.

**Review by mth-1012 — APPROVED** [review]
I made some small changes that I felt were necessary. Those are ready to be merged to covidForms branch and to master.

File: frontend/src/components/Tabs/AdminTabs/RoleRequest.js [hunk 2]
```diff
@@ -12,12 +12,12 @@ function RoleRequest(props) {
...
-  const [immigrationOfficerList, setIOList] = useState([]);
- 
+  const [immigrationOfficialList, setIOList] = useState([]);
 
-  useEffect(() => {
+
+  useEffect(async () => {
     console.log("Patient List is Loaded!");
-    axios.get(`admin/${currentUser.user.uid}/pending-patients`)
+    await axios.get(`admin/${currentUser.user.uid}/pending-patients`)
```
[the remark this answers is not in the document]
Comment (mth-1012) [reply frontend/src/components/Tabs/AdminTabs/RoleRequest.js]: You're right. I forgot to remove them when I thought they were the solutions. Removed now.

File: frontend/src/components/Tabs/AdminTabs/RoleRequest.js [hunk 3]
```diff
@@ -15,128 +15,142 @@ function RoleRequest(props) {
...
+      })
+  }, []);
+
+  // function renderPendingPatientList() {
+  //   if (patientList === null) return;
+  //   return <div>
+  //     {patientList.map((patient, index) => <RoleRequestBox
+  //       key={index}
+  //       RequesterUsername={patient.name}
+  //       RequestType={patient.userType}
```
Comment (AC159) [inline frontend/src/components/Tabs/AdminTabs/RoleRequest.js:63]: Please remove these commented lines. Commented code should not appear in a pull request

**Review by p-carrier — APPROVED** [review]
I think we covered pretty much everything.

**Pull request merged**